.ci_support/linux_64_.yaml

```diff
@@ -1,7 +1,7 @@
 aws_crt_cpp:
 - 0.29.9
 aws_sdk_cpp:
-- 1.11.488
+- 1.11.458
```

Comment:
Why is it downgrading the aws-sdk-cpp pin? Need to investigate

Reply:
Ok, this is expected behavior. The aws-sdk-cpp 1.11.488 migration was rolled back because it broke arrow

xref: https://github.com/conda-forge/tiledb-feedstock/pull/412#discussion_r1923952905